packages/test/test-end-to-end-tests/src/test/data-virtualization/groupIdInSummary.spec.ts

```diff
@@ -313,6 +314,95 @@ describeCompat(
 			}
 		});
 
+		it("Can create loadingGroupId without feature", async function () {
```

Comment:
Can you change the name of test to actually represent what it is testing? 
Like: Loading Snapshopt with GroupId using feature gate off should load properly?